src/i18n/strings/en_EN.json

```diff
@@ -2554,6 +2557,8 @@
                 "session_key": "Session key:",
                 "title": "Advanced"
             },
+            "confirm_key_storage_off": "Are you sure you want to keep key storage turned off?",
+            "confirm_key_storage_off_description": "If you sign out of all your devices you will lose your message history and will need to verify all your existing contacts again. <a>Learn more</a>",
```

Comment:
```suggestion
            "confirm_key_storage_off_description": "If you sign out of all your devices, you will lose your message history and will need to verify all your existing contacts again. <a>Learn more</a>",
```

possibly?

Reply:
This is direct from the designs, so I think we should leave it as-is.

Reply:
I mean, the designs are wrong, but fine :)